### PR TITLE
make the access-granting script work on group-like accounts

### PR DESCRIPTION
**Pull request by SHuang-Broad**

### CODE REVIEW

File: scripts/misc/grant_access_to_collab.sh [hunk 1]
```diff
@@ -46,14 +47,18 @@ while [ $# -ge 1 ]; do
...
             else
                 throw_error "--user requires a non-empty argument"
             fi
             ;;
         --user=?*)
-            user_email=${1#*=} # remove everything up to = and assign rest to user
+            account_email=${1#*=} # remove everything up to = and assign rest to user
+            shift
+            ;;
+        -g|--group)
```
Comment (bshifaw) [inline scripts/misc/grant_access_to_collab.sh:60]: The group parameter may need to be added to the help message

File: scripts/misc/grant_access_to_collab.sh [hunk 2]
```diff
@@ -129,10 +134,11 @@ prefix="${prefix%/}"
...
 do
-    echo -e "\nGranting user ${user_email} with ${access_level}-level access to file type ${t} under gs://${gcs_bucket}/${prefix}/"
-    gsutil -mq acl \
-        ch -u "${user_email}":"${access_level}" \
-        -R "gs://${gcs_bucket}/${prefix}/**${t}"
+    echo -e "\nGranting ${account_email} with ${access_level}-level access to file type ${t} under gs://${gcs_bucket}/${prefix}/"
+    gsutil -mq acl ch \
+        "${account_type}" \
+        "${account_email}":"${access_level}" \
+        -R "gs://${gcs_bucket}/${prefix}/**/*${t}"
```
Comment (bshifaw) [inline scripts/misc/grant_access_to_collab.sh:141]: why the extra `/*` ?

**Review by bshifaw — CHANGES_REQUESTED** [review]
Minor update to the help message

File: scripts/misc/grant_access_to_collab.sh (hunk 2 above)
Comment (SHuang-Broad) [reply scripts/misc/grant_access_to_collab.sh:141]: That double star is for any "sub-directories".
That single star is for globing the file with the right extension.

I've tested this by checking for BAMs that are two levels below `prefix`, and immediately under `prefix`.

**Review by bshifaw — APPROVED** [review]
👍

**Pull request merged**